packages/client/src/components/formConfig/Canvas.tsx

```diff
@@ -100,7 +140,10 @@ export function Canvas({
   const fieldsMap = useSelector((store: IStoreState) =>
     selectConfigFields(store, event, section)
   )
-  const configFields = generateConfigFields(fieldsMap)
+  const formSection = useSelector((store: IStoreState) =>
+    getRegisterFormSection(store, section, event)
```

Comment:
We will need to use `selectConfigRegisterForm` instead as the dependecy on register reducer was removed in https://github.com/opencrvs/opencrvs-core/pull/2780/commits/c9544116baede952e584dd75edb4f284273f8a32